README.md

```diff
@@ -1,31 +1,312 @@
+```
+./beacon-simulator.sh <ip> <port> <interval> <jitter> <protocol> <max payload>
+```
+`ip`: The IP address of the server.</br>
+`port`: The server port listening for the connection. (Or the ICMP type: 8 = `ping`, 13 = `timestampreq`, 17 = `addrmaskreq`)</br>
+`interval`: The amount of time in seconds between beacons.</br>
+`jitter`: The amount of jitter in seconds.</br>
+`protocol` (optional): The protocol of the beacon. Accepts `tcp`, `udp`, or `icmp`. Defaults to `tcp`.</br>
+`max payload` (optional): The beacon's maximum payload size in bytes. The payload is a random string of a's ranging from 0 to the maximum payload size. Defaults to `1424`.</br>
+
+### Example Commands
+```
+./beacon-simulator.sh 192.168.56.104 9000 30 5
+```
+Sends a random payload of up to 1424 bytes to port 9000 on the device at IP 192.168.56.104 every 25-35 seconds using the TCP protocol.
+
+```
+./beacon-simulator.sh 192.168.56.101 3333 300 8 udp 256
+```
+Sends a random payload of up to 256 bytes to port 3333 on the device at IP 192.168.56.101 every 292-308 seconds using the UDP protocol.
+
+```
+./beacon-simulator.sh 192.168.56.102 8 10 2 icmp
+```
+Pings the server at 192.168.56.102 every 8-12 seconds.
+
+<p align="right"><a href="#table-of-contents">Back to Top</a></p>
+
+</br>
+</br>
+
+## `simple-beacon.sh`
+`simple-beacon.sh` should run on the machine simulating the compromised client. It sends a very simple HTTP request with a custom user-agent string (`Modzilla/0.0001(Atari7800)`) to a specified IP address or FQDN every 200-350 seconds. This script can be used on its own without any additional scripts such as `simple-listener.sh`.
+
+### Dependencies
+This script uses common utilities commonly pre-installed on Unix-like systems. You should not need to install them. For your reference, the required utilities are:
+
+- `curl`
+- `shuf`
+- `sleep`
+
+#### Command Syntax
+```
+./simple-beacon.sh <IP or FQDN>
+```
+`IP or FQDN`: The IP address or FQDN of the server.
+
+#### Example Commands
+```
+./simple-beacon.sh activecountermeasures.com
+```
+Simulates a jittered beacon by sending HTTP requests to activecountermeasures.com
+
+<p align="right"><a href="#table-of-contents">Back to Top</a></p>
+
+</br>
+</br>
+
+## Python Script Pairs
+The scripts within the `python-scripts` directory work in pairs. `tcp-client.py` works with `tcp-server.py`, and `udp-client.py` works with `udp-server.py`. Unlike the other threat tools, these scripts can send a beacon to more than one server. Both client scripts will require editing the code so the IP addresses match your server IP(s). You can also edit the code to configure the server port for the C2 channel, the payload size, and the beaconing interval if you disagree with the defaults.
+
+Default Server Port: 9000</br>
+Default Payload Size: 0 - 1200 bytes</br>
+Default Beaconing Interval: 30 - 60 seconds
+
+### Dependencies
+This script will require the following tools be installed on the system running the script:
```

Comment:
We need to make it clear that these dependencies must be installed on both systems/ for both scripts.